intelmq/bots/experts/idea/expert.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+from collections import Sequence, Mapping
+from base64 import b64decode
+from uuid import uuid4
+from urllib.parse import quote_plus
+
+from intelmq.lib.bot import Bot
+
+
+def quot(s):
```

Comment:
You sure? This is also helper function for translation definition, deliberately short (because used on various places), and deliberately misnamed to not get confused with urllib.parse.quote.